attest/activation.go

```diff
@@ -0,0 +1,273 @@
+package attest
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"errors"
+	"fmt"
+	"io"
+
+	tpm1 "github.com/google/go-tpm/tpm"
+	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpm2/credactivation"
```

Comment:
Maybe a good TODO to move credactivation and verification to an internal package for go-attestation?

Reply:
Done.